base/features/asciidoctor/skeleton/gradle-build/gradle/asciidoc.gradle

```diff
@@ -0,0 +1,24 @@
+asciidoctorj {
+    version '2.0.0'
+    modules{
+        diagram{
+            version '1.5.16'
+        }
+    }
+
+    options doctype: 'book', ruby:'erubis'
+
+    attributes 'sourcedir': 'src/docs/asciidoc',
+            'source-highlighter': 'coderay',
+            toc: 'left',
+            idprefix: '',
+            idseparator: '-',
+            'icons': 'font',
+            'setanchors': '',
+            'listing-caption':'',
+            'imagesdir': 'images',
```

Comment:
I think only attributes with '-' need to be surrounded by quotes for Groovy syntax.
Also some attributes below use double quotes. Imho :% for consistency, all should use the same syntax.